### ISSUE TITLE
Crashlytics install/setup not showing on firebase dashboard

### ISSUE THREAD

**Issue by Aceman18** [EddyVerbruggen/nativescript-plugin-firebase#1113]
Hi, 
I have been trying to add crashlytics to my app, I've followed the guide in the docs folder. 

However when I go onto my firebase dashboard I’m still stuck with the Crashlytics install wizard. 
![screen shot](https://user-images.githubusercontent.com/24454702/51331604-068cde80-1a72-11e9-9c66-f7370bd6629d.png)

I've tried following the firebase process by installing Fabric and working through that process but it never manage to get it to detect the app?
![fabric](https://user-images.githubusercontent.com/24454702/51332269-53bd8000-1a73-11e9-92f7-85368d773e85.png)

I've looked around and found a forum post from last year (https://discourse.nativescript.org/t/adding-crashlytics-to-nativescript-app/5843) with a similar issue that didn't seem to get resolved.

I'm not sure whether to follow the docs here or try to get Fabric working? and if I follow the docs here what do I do about the install wizard in the firebase console?

Thanks for your help!

**Comment by EddyVerbruggen** [comment]
AFAIK you need to select "This app is new to Crashlytics", and press "Next". Then the screen changes to something like "waiting for your first crash report". Then produce a crash, and it can easily take a day before that screen changes (later data comes in much quicker).

**Comment by Aceman18** [comment]
Ok great, I've done that and it tells you to go to the docs:
![screen shot 2019-01-17 at 16 55 49](https://user-images.githubusercontent.com/24454702/51334975-d7c63680-1a78-11e9-8a69-f9f22ad4a5c0.png)

Previously when I tried that option I never clicked on the link to go to the docs as I already had it open in another tab in my browser. 

However, after clicking the link it takes you onto the third step that i never saw:
![screen shot 2019-01-17 at 16 55 55](https://user-images.githubusercontent.com/24454702/51335099-25db3a00-1a79-11e9-8a37-fd72fc87bee7.png)

My mistake for not clicking that button and just going to another tab! 

Thanks for your help, I've added some code that forces an error so hopefully I will see that in the next day or so.
